source/dub/dub.d

```diff
@@ -422,6 +422,29 @@ class Dub {
 		write(dst.toPrettyString());
 	}
 
+	private void listPaths(string methodName)(BuildPlatform platform, string config)
+	{
+		string[] importPaths;
+
+		__traits(getMember, m_project, methodName)(importPaths, platform, config);
```

Comment:
Can do. I was just following the way the other methods in the class were using output parameters.